de.metas.serviceprovider/src/main/sql/postgresql/system/45-de.metas.serviceprovider/5558160_sys_gh5816-app_update_unique_indexes_to_consider_isActive.sql

```diff
@@ -0,0 +1,70 @@
+-- 2020-04-28T15:19:02.739Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Index_Table SET WhereClause='S_Issue.IssueURL IS NOT NULL and S_Issue.IsActive=''Y''',Updated=TO_TIMESTAMP('2020-04-28 18:19:02','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Index_Table_ID=540527
```

Comment:
```suggestion
UPDATE AD_Index_Table SET WhereClause='S_Issue.IsActive=''Y''',Updated=TO_TIMESTAMP('2020-04-28 18:19:02','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Index_Table_ID=540527
```

`S_Issue.IssueURL IS NOT NULL` does not break things, but also doesn't help and is therefore useless

Background In the postgresql world, `NULL` means "unknown", and 
* `NULL != NULL`
* therefore two different `S_Issues` can have their `IssueURL` being NULL without violating the unique contraint
* (sidenote: in fact, *if* you needed to prevent this, you would need to have the unique-index on something like `COALESCE(IssueURL,'') )